lm-api/tests/api/routes/test_cluster_statuses.py

```diff
@@ -0,0 +1,159 @@
+from fastapi import status
+from httpx import AsyncClient
+from pytest import mark
+from sqlalchemy import select
+
+from lm_api.api.models.cluster_status import ClusterStatus
+from lm_api.permissions import Permissions
+
+
+@mark.parametrize(
+    "permission",
+    [
+        Permissions.STATUS_UPDATE,
+        Permissions.ADMIN,
+    ],
+)
+@mark.asyncio
+async def test_add_cluster_status(
+    permission, backend_client: AsyncClient, inject_security_header, read_object
+):
+    """Test adding a new cluster status."""
+    cluster_client_id = "cluster_1"
+    interval = 60
+
+    inject_security_header("owner1@test.com", permission, client_id=cluster_client_id)
+
+    response = await backend_client.put("/lm/cluster_statuses", params={"interval": interval})
+    assert response.status_code == status.HTTP_202_ACCEPTED
+
+    stmt = select(ClusterStatus).where(ClusterStatus.cluster_client_id == cluster_client_id)
+    cluster_status_fetched = await read_object(stmt)
+
+    assert cluster_status_fetched.cluster_client_id == cluster_client_id
+    assert cluster_status_fetched.interval == interval
+
+
+@mark.parametrize(
+    "permission",
+    [
+        Permissions.STATUS_UPDATE,
+        Permissions.ADMIN,
+    ],
+)
+@mark.asyncio
+async def test_update_cluster_status(
+    permission,
+    backend_client: AsyncClient,
+    inject_security_header,
+    create_one_cluster_status,
+    read_object,
+):
+    """Test updating an existing cluster status."""
+    cluster_client_id = "dummy_1"
+    interval = 120
+
+    inject_security_header("owner1@test.com", permission, client_id=cluster_client_id)
+
+    response = await backend_client.put("/lm/cluster_statuses", params={"interval": interval})
+    assert response.status_code == status.HTTP_202_ACCEPTED
```

Comment:
This test should check the last updated value, shouldn't it?

Reply:
Ideally, you would freeze the time and make sure that the last_updated value matches the frozen time.

Reply:
I updated the test to use the new `freeze` feature from `pendulum.test`.